src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -2556,7 +2556,7 @@
     <value>Run selected PowerShell script</value>
   </data>
   <data name="LaunchPreviewPopupDescription" xml:space="preserve">
-    <value>Launch preview in popup window</value>
+    <value>Show file preview in a popup window</value>
```

Comment:
This action works for folders as well, perhaps 'item' would be more appropriate.